intermine/intermine-ws-python
# Query manager

Hi @julie-sullivan @rachellyne ,
This pull request adds a file `query_manager.py` ( other than the registry features )
This file contains the following functions:
`get_all_query_names()` to view the saved queries from a user account ,
 `get_query()` to fetch information about a saved query,
 `delete_query()` to delete a saved query from user account
`post_query()` to post a query to user account

Steps to test:

- git clone https://github.com/nupurgunwant/intermine-ws-python.git
- cd intermine-ws-python
- python setup.py install
- pip install -r requirements.txt
- git checkout query_manager
- python(this will open the interactive mode of python, make sure the version is >=3)
==>from intermine import query_manager as qm
'Enter the account's API token and mine to which account is linked'
==>qm.get_all_query_names()
'returns the names of saved queries'
==>qm.get_query()
'enter name of query you want details about'
'returns details'
==>qm.delete_query()
'enter name of query you want to delete'
==>qm.post_query()
'enter the xml of the query you want to save on the account'

You may check your account after each function to check if the results are correct. Looking forward to all the suggestions. :)
Here's a screenshot in reference to the above steps.
<img width="1435" alt="screen shot 2018-05-16 at 11 20 32 am" src="https://user-images.githubusercontent.com/19436036/40111680-7e643fb8-58fb-11e8-90c6-ed4fe598a983.png">

PS : Merging this PR will add registry features as well.
